simpegEM/FDEM/FDEM.py

```diff
@@ -221,8 +322,9 @@ def getA(self, freq):
         mui = self.MfMui
         sigI = self.MeSigmaI
         C = self.mesh.edgeCurl
+        iomega = 1j * omega(freq) * sp.eye(self.mesh.nF)
 
-        return mui*C*sigI*C.T*mui + 1j*omega(freq)*mui
+        return C*sigI*C.T*mui + iomega
```

Comment:
This was for symmetry! so we actually might want an option to keep this in. And potentially have symmetry the default? I am up for changing this though....